### ISSUE TITLE
AttributeError: module 'pyarrow' has no attribute 'Column'

### ISSUE THREAD

**Issue by dazzag24** [cldellow/csv2parquet#12]
Thanks for this useful tool!

Not sure if this an error or I am doing something wrong?

Created an new virualenv
`pipenv shell`

Installed requirements
`pipenv install --skip-lock pyarrow csv2parquet`

Attempted to convert simple csv using simplest invocation as I want all rows and columns
```
> csv2parquet a.csv
Traceback (most recent call last):
  File "/home/xxx/.local/share/virtualenvs/csv2parquet-tyfa8dfH/bin/csv2parquet", line 8, in <module>
    sys.exit(main())
  File "/home/xxx/.local/share/virtualenvs/csv2parquet-tyfa8dfH/lib/python3.6/site-packages/csv2parquet/csv2parquet.py", line 247, in main
    main_with_args(convert, sys.argv[1:])
  File "/home/xxx/.local/share/virtualenvs/csv2parquet-tyfa8dfH/lib/python3.6/site-packages/csv2parquet/csv2parquet.py", line 244, in main_with_args
    args.type)
  File "/home/xxx/.local/share/virtualenvs/csv2parquet-tyfa8dfH/lib/python3.6/site-packages/csv2parquet/csv2parquet.py", line 157, in convert
    for x in range(len(fields)) if keep[x]]
  File "/home/xxx/.local/share/virtualenvs/csv2parquet-tyfa8dfH/lib/python3.6/site-packages/csv2parquet/csv2parquet.py", line 157, in <listcomp>
    for x in range(len(fields)) if keep[x]]
AttributeError: module 'pyarrow' has no attribute 'Column'
```

`python --version
Python 3.6.8`

more Pipfile
```
[[source]]
name = "pypi"
url = "https://pypi.org/simple"
verify_ssl = true

[dev-packages]

[packages]
pyarrow = "*"
csv2parquet = "*"

[requires]
python_version = "3.6"```

**Comment by dazzag24** [comment]
I think that "Column" has been removed in the 0.15 release of PyArrow.  

https://github.com/apache/arrow/pull/4841
https://github.com/apache/arrow/commit/c350bbaaa9b1155b9f54d4b312abc8208e13bea7

**Comment by cldellow** [comment]
D'oh, thanks for the bug report and pointers!

I might have time to look at patching this to use the new interfaces this weekend, or failing that, the next. Someone submitted [a PR for another project of mine to upgrade it](https://github.com/cldellow/sqlite-parquet-vtable/pull/39), and it doesn't look too big of a change. 

If you need to get unblocked in the interim, you could clone the repo and patch the Pipfile to use a specific, older version of pyarrow.

**Comment by cldellow** [comment]
Hey @dazzag24, thanks again for the report!

This should be fixed by https://github.com/cldellow/csv2parquet/commit/2e21226df2de343b84412d72f2f77aed1f3210ac, which is now published to pypi as v0.0.8.

**Comment by dazzag24** [comment]
Many thanks!

I have tested and can confirm that this now works.
